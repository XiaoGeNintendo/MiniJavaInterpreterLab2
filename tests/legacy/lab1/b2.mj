int main(){
    char x='a';
    char y='z';
    int z2=x*y;
    int what='a'*'z';
    string s="asd";
    string sm="as";
    string t=sm+"d";
    boolean equal=s==t;

    println(x);
    println(y);
    println(z2);
    println(what);
    println(equal);
    return 0;
}