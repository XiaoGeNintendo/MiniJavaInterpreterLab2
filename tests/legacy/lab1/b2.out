a
z
11834
11834
true
Process exits with the code 0.